Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -778,16 +778,6 @@ export class BotProject {
 
     for (const file of files) {
       const { realFilePath, content } = file;
-      await this.fileStorage.removeFile(realFilePath);
```

Comment:
Remove this dangerous delete and write back is ok, but i think we can take the chance to totally disable this whole "tranformer" work for now,  which is done somewhat very ad-hoc. 

One top thing in R10 is about version, we should take the transformer work with that to have a better transformer. 

What do you think @cwhitten

Reply:
I agree remove  those two "transformer" methods, they already exist about 2 month on master, user's bot should already be migrated.